.azure-pipelines/testbed_health_check.py

```diff
@@ -208,6 +214,7 @@ def pre_check(self):
         if len(ipv4_not_exists_hosts) > 0:
             raise HostsUnreachable(self.check_result.errmsg)
 
+        # TODO: Refactor the following code to specify a "leader" T2 Testbed and skip the check on "followers"
         # Retrieve the basic facts of the DUTs
         duts_basic_facts = self.sonichosts.dut_basic_facts()
```

Comment:
Can directly use `self.duts_basic_facts` because it was initialized in `def init_hosts(self):`